include/seastar/util/log.hh

```diff
@@ -87,7 +87,7 @@ class logger {
     static std::ostream* _out;
     static std::atomic<bool> _ostream;
     static std::atomic<bool> _syslog;
-    static inline thread_local bool silent = false;
+    static inline thread_local std::atomic<bool> silent = false;
```

Comment:
Why this change? It's thread local, it doesn't need to be atomic.

Reply:
sorry, i forget to revert the leftovers of my experiments. reverted.